UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.API.FunctionalTests/Helpers/GetBatchDetails.cs

```diff
@@ -39,30 +42,35 @@ public static void GetBatchDetailsResponseValidationForFullAVCSExchangeSet(dynam
             if (mediaType.Equals("Zip"))
             {
                 string fileName = batchDetailsResponse.files[0].filename;
-                if (fileName.Equals($"{posDetails.UpdateExchangeSet}"))
+                if (fileName.Equals(string.Format(posDetails.PosUpdateZipFileName, weekNumber, currentYear)))
                 {
-                    Assert.That(fileName, Is.EqualTo($"{posDetails.UpdateExchangeSet}"), $"Expected Response File Name Zip of {posDetails.UpdateExchangeSet}, but actual value is {fileName}");
+                    Assert.That(fileName, Is.EqualTo(string.Format(posDetails.PosUpdateZipFileName, weekNumber, currentYear)), $"Expected Response File Name Zip of {posDetails.UpdateExchangeSet}, but actual value is {fileName}");
```

Comment:
Can we use string interpolation please in this whole class wherever feasible.

Reply:
Not feasible because we are altering the filename after collecting it from appsettings.